spec/features/social_features_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+feature "Social networking" do
+    given(:jane) { Fabricate(:user) }
+    given(:joe) { Fabricate(:user) }
+    given(:tv_comedies) { Fabricate(:category, name: 'TV Comedies') }
+
+  scenario "following a user" do 
+    futurama = Fabricate(:video, title: 'Futurama', category: tv_comedies)
+    jane_review = Fabricate(:review, video: futurama, user: jane)
+    
+    sign_in(joe)
+    find("a[href='/videos/#{futurama.id}']").click 
+    click_on jane_review.user.full_name
+    click_on 'Follow'
+    expect(current_path).to eq(people_path)
```

Comment:
Gotcha and great explanation! Not sure if you saw my reply to one of your previous comments (my additional commit may have hidden it...) but this is what I was trying to clarify. My understanding is this - In the spec, my use of the expect(page) assertion should be used on the "landmarks" (elements that should exist on a page) without being too closely tied to specifics that could change ( ex., the peeling paint gets redone, or scarecrow could get a facelift). I started second guessing myself a bit after your initial comments on usage of expect(page.